content/posts/May-2020.md

```diff
@@ -4,6 +4,40 @@ date: 2020-05-06
 draft: false
 ---
 
+## Week 22 (May 25th - May 29th)
+
+### packit
+
+* [The problem with kerberos initialization](https://github.com/packit-service/packit/issues/837)
+  was fixed. ([#838])
+* We released a new version of the packit package. ([release 0.11.1])
+* We have been also working on some smaller issues and preparation steps for our future work.
+  ([#841], [#843], [#846], [#847])
+
+[#838]: https://github.com/packit-service/packit/pull/838
+
+[#841]: https://github.com/packit-service/packit/pull/841
+[#843]: https://github.com/packit-service/packit/pull/843
+[#846]: https://github.com/packit-service/packit/pull/846
+[#847]: https://github.com/packit-service/packit/pull/847
+[release 0.11.1]: https://github.com/packit-service/packit/releases/tag/0.11.0
+
+### packit-service
+
+* [Jirka] implemented creating new bug in Bugzilla for CentOS-Stream pull-requests ([#627]).
+  This will be triggered by the added label in the (near) future.
```

Comment:
As it's not implemented, I would remove this.

Reply:
This goes against what Tomáš wrote above. We want to have users in the loop.